eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -4,19 +4,19 @@ parameters:
   Matrix:
     Linux_Node8:
       OSVmImage: "ubuntu-18.04"
-      NodeVersion: "8.x"
+      NodeTestVersion: "8.x"
```

Comment:
where does it give the version of node that is running the build?

Reply:
https://github.com/Azure/azure-sdk-for-js/blob/84b3b773805418ad06b711f5bdd54c1da5aa32e8/eng/pipelines/templates/variables/globals.yml#L3

Reply:
Our pipelines are factored to reduce duplication which means you often need to search to find things.